File: desafio-02/alexsandro-matias/python/desafio-02.py
def verifica_primo(numero):
    contador = 0
    for i in range (numero):
        i = i + 1
        if numero % i == 0:
            contador = contador + 1

    if contador == 2:
        print(numero)

for i in range(10000):
    verifica_primo(i)

File: desafio-02/alexsandro-matias/java/Desafio-02.java
package programadores;

public class Desafio02
{
	public static void main(String[] args)
	{
		for (int j = 2; j < 1000; j++)
		{
			int contador = 0;
			for (int i = 1; i <= j; i++)
			{
				if ((j % i) == 0)
				{
					contador++;
				}
			}
			if (contador == 2)
			{
				System.out.println(j);
				j++;
			}
		}
	}
}
